### PR TITLE
fix: improve Avneesh's profile

### PR DESCRIPTION
**Pull request by krishguptadev**
Fixed all typos.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/750"></a>

### CODE REVIEW

**Comment by avneesh0612** [comment]
Lgtm 😂

**Review by schmelto — APPROVED** [review]

**Review by avneesh0612 — APPROVED** [review]

**Review by avneesh0612 — COMMENTED** [review]
You formatted someone else's lol

**Comment by krishguptadev** [comment]
> You formatted someone else's lol

It's a bug, being fixed in #748 but ig the action will make me fix it in this PR itself :joy:

**Comment by avneesh0612** [comment]
Haha lol

File: public/data/avneesh0612.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Avneesh Agarwal",
-  "bio": "I am a web developer, designer, content creator, open source contributor, and student.",
+  "bio": "I am a web developer, designer, content creator, open source contributor, and a student.",
```
Comment (Panquesito7) [inline public/data/avneesh0612.json:3]: Not sure if this `a` is needed as there's already an `a` at the beginning. 🤔

Comment (krishguptadev) [reply public/data/avneesh0612.json:3]: Avneesh asked be to add it tho 😓

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**